### DISCUSSION TITLE
How to use dropbar?

### DISCUSSION THREAD

**Discussion by boydkelly** [stevearc/dressing.nvim#161] [Q&A] [answered]
Dropbar.nvim says it can be used with vim.ui.select.   Can this be backend for dressing config?   How can this be integrated?
Thanks for any ideas!

https://github.com/Bekaboo/dropbar.nvim?tab=readme-ov-file#usage-with-vimuiselect

**Comment by stevearc** [comment, marked as answer]
You don't _really_ need dressing if you're going to use another plugin for the `vim.ui.select` implementation. Just follow the instructions for dropbar, and then disable dressing for select

```lua
require("dressing").setup({
  select = {
    enabled = false
  }
})
```

If you or someone else is interested, dropbar could be added as a supported backend (like fzf-lua) to make things a tiny bit more convenient.

**Reply by boydkelly** [reply]
Thanks!!

**Reply by willothy** [reply]
> If you or someone else is interested, dropbar could be added as a supported backend (like fzf-lua) to make things a tiny bit more convenient.

I wrote the `vim.ui.select` integration for dropbar - would be happy to PR a backend for Dropbar in here!

I am quite busy with work lately though, @stevearc if I create an issue would you mind assigning me so I have a reminder to get to this when I have time? Thanks :)
